core/src/test/java/org/apache/calcite/test/MaterializedViewSubstitutionVisitorTest.java

```diff
@@ -410,6 +410,21 @@
     sql(mv, query).noMat();
   }
 
+  /**
+   * stddev_pop aggregate function does not support roll up.
+   */
+  @Test public void testSqlAggFunctionRollup() {
```

Comment:
In current `MaterializedViewSubstitutionVisitorTest.java`, there's already an `testAggregateRollUp`. How about rename this as `testSqlAggFunctionRollup2` and put it after `testAggregateRollUp` ?